tools/minikube/scripts/start_pods.sh

```diff
@@ -53,6 +68,10 @@ if kubectl get configmap --namespace=catalog ansible-controller-env &>/dev/null;
 	kubectl delete --namespace=catalog configmap ansible-controller-env
 fi
 
+if kubectl get configmap --namespace=catalog ansible-insights-env &>/dev/null; then
```

Comment:
@hsong-rh Can this also be removed, since we just have a single boolean settings attribute flowing thru.